registration-processor/registration-processor-landing-zone-utility/src/main/java/io/mosip/registration/processor/util/LandingZoneUtility.java

```diff
@@ -4,6 +4,7 @@
 import java.io.File;
 import java.io.FileInputStream;
 import java.io.FileNotFoundException;
+import java.io.IOException;
```

Comment:
is this class contains any bug changes ?

Reply:
yes at line no 86 catch block is added to catch IOException , Since we have used java.nio.file.Files at line no 72